### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add `README.yaml`
* Add `.travis.yaml`
* Add related projects

## why
* Maintenance
* Discoverability

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+SHELL := /bin/bash
+
+# List of targets the `readme` target should call before generating the readme
+export README_DEPS ?= docs/targets.md docs/terraform.md
```
Comment (aknysh) [inline Makefile:4]: why the Markdown docs are targets or dependencies?

Comment (osterman) [reply Makefile:4]: this is classic make. In fact, historically, targets were usually files or globs. 

https://github.com/cloudposse/build-harness/blob/master/modules/docs/Makefile#L5-L18

image

this says build `hello` if the binary does not exist. But before you can build `hello`, build `main.o`, etc...

File: docs/terraform.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| disk_queue_depth_threshold | The maximum number of outstanding IOs (read/write requests) waiting to access the disk. | string | `64` | no |
+| free_storage_space_threshold | The minimum amount of available storage space in Byte. | string | `2000000000` | no |
+| freeable_memory_threshold | The minimum amount of available random access memory in Byte. | string | `64000000` | no |
+| swap_usage_threshold | The maximum amount of swap space used on the DB instance in Byte. | string | `256000000` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| sns_topic_arn |  |
```
Comment (sarkis) [inline docs/terraform.md]: missing description for output

File: README.md [hunk 3]
```diff
@@ -26,24 +53,170 @@ resource "aws_db_instance" "default" {
...
+| disk_queue_depth_threshold | The maximum number of outstanding IOs (read/write requests) waiting to access the disk. | string | `64` | no |
+| free_storage_space_threshold | The minimum amount of available storage space in Byte. | string | `2000000000` | no |
+| freeable_memory_threshold | The minimum amount of available random access memory in Byte. | string | `64000000` | no |
+| swap_usage_threshold | The maximum amount of swap space used on the DB instance in Byte. | string | `256000000` | no |
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| sns_topic_arn |  |
```
Comment (sarkis) [inline README.md]: missing description for output

**Review by sarkis — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -26,24 +53,170 @@ resource "aws_db_instance" "default" {
...
   parameter_group_name = "default.mysql5.7"
-  apply_immediately = "true"
-  skip_final_snapshot = "true"
+  apply_immediately    = "true"
+  skip_final_snapshot  = "true"
 }
 
 module "rds_alarms" {
-  source = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
+  source         = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
```
Comment (aknysh) [inline README.md]: update `source`

File: README.md [hunk 5]
```diff
@@ -26,24 +53,170 @@ resource "aws_db_instance" "default" {
...
+
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|:----:|:-----:|:-----:|
+| burst_balance_threshold | The minimum percent of General Purpose SSD (gp2) burst-bucket I/O credits available. | string | `20` | no |
+| cpu_credit_balance_threshold | The minimum number of CPU credits (t2 instances only) available. | string | `20` | no |
+| cpu_utilization_threshold | The maximum percentage of CPU utilization. | string | `80` | no |
+| db_instance_id | The instance ID of the RDS database instance that you want to monitor. | string | - | yes |
```
Comment (aknysh) [inline README.md:79]: use `` (instead of `-`) as default empty string?

File: README.yaml [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+    name                 = "mydb"
+    username             = "foo"
+    password             = "foobarbaz"
+    parameter_group_name = "default.mysql5.7"
+    apply_immediately    = "true"
+    skip_final_snapshot  = "true"
+  }
+
+  module "rds_alarms" {
+    source         = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
```
Comment (aknysh) [inline README.yaml]: update `source`

**Review by aknysh — COMMENTED** [review]
inline comments

File: README.md (hunk 5 above)
Comment (osterman) [reply README.md:79]: `terraform-docs` writes the `-`

**Review by aknysh — APPROVED** [review]

**Pull request merged**